community/numix-themes/APKBUILD

```diff
@@ -1,55 +1,45 @@
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=numix-themes
 _pkgname=numix-gtk-theme
-pkgver=2.6.1
+pkgver=2.6.4
 pkgrel=0
 pkgdesc="A modern flat theme with a combination of light and dark elements, GTK 2 and 3"
 url="http://shimmerproject.org/project/Numix/"
 arch="noarch"
 license="GPLv3"
-makedepends="$depends_dev"
 subpackages="$pkgname-gtk2 $pkgname-gtk3 $pkgname-metacity $pkgname-xfwm4 $pkgname-xfce4-notifyd:_notify
 		$pkgname-openbox:_openbox"
 source="numix-$pkgver.tar.gz::https://github.com/numixproject/$_pkgname/archive/$pkgver.tar.gz"
-
-_builddir="$srcdir"/$_pkgname-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+builddir="$srcdir"/$_pkgname-$pkgver/src
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
 	# Remove Unity theme
 	rm -f gtk-3.0/apps/unity.css
 	sed -i '/unity\.css/d' gtk-3.0/gtk.css
 	rm -rf unity
 }
 
 package() {
-	cd "$_builddir"
+	cd "$builddir"
 	mkdir -p "$pkgdir"/usr/share/themes/Numix
-	cp -pr * "$pkgdir"/usr/share/themes/Numix
-	rm -f "$pkgdir"/usr/share/themes/Numix/Makefile
+	cp -pr ../* "$pkgdir"/usr/share/themes/Numix
+	cd "$pkgdir"/usr/share/themes/Numix
+	rm -rf src/assets scripts Makefile
 }
 
 _mv() {
 	pkgdesc="$1"
 	install_if="$pkgname=$pkgver-r$pkgrel $2";
-	depends=
 	local _i
 	shift; shift
 	mkdir -p "$subpkgdir"/usr/share/themes/Numix
 	for _i; do
 		msg "moving $_i"
-		mv "$pkgdir"/usr/share/themes/Numix/$_i  \
+		mv "$pkgdir"/usr/share/themes/Numix/src/$_i  \
 			"$subpkgdir"/usr/share/themes/Numix || return 1
 	done
+	if rmdir "$pkgdir"/usr/share/themes/Numix/src 2>/dev/null; then return 0; fi
```

Comment:
using `rmdir` on it's own would be detected as an error by `abuild` if it failed (which it would several times until everything was moved).

enclosing `rmdir` in an `if` lets the package build.